extensions/warp-ipfs/src/thumbnail.rs

```diff
@@ -44,13 +46,23 @@ impl Default for ThumbnailId {
     }
 }
 
-#[derive(Default, Clone)]
+#[derive(Clone)]
 #[allow(clippy::type_complexity)]
```

Comment:
Not related to your PR but IMO it's best to avoid ignoring clippy lints as much as possible. This one for example, is just asking for a simple type alias which would make the code easier to read/follow and so its good advice.

Reply:
It is good advice. This was more of me ignoring the lint and forgot about it since I didnt view the type as hard to read, but I did update it in this PR just as a small cleanup.